### DISCUSSION TITLE
Please let me ignore notifications from certain repos/projects

### DISCUSSION THREAD

**Discussion by twig** [community/community#112422] [Other Feature Feedback, Questions, & Ideas] [answered]
I keep getting notifications from https://github.com/twigphp/Twig because they use `@twig` as part of the syntax 

As lovely as the people here are, I'm really not interested.

**Comment by AidenJamesYt** [comment]
heres what i found out:

Navigate to Repository Settings:

Visit the repository page on GitHub whose notifications you want to ignore (it's https://github.com/twigphp/Twig).
Click on the "Settings" tab located near the top-right corner of the repository page.
Scroll down until you find the "Danger Zone" section.
Click on "Ignore this repository" within the "Danger Zone" section.

**Reply by twig** [reply, marked as answer]
Ah that didn't quite work for me, couldn't find danger zone

But I did find the "ignore" option in the repository "watch" dropdown which should do the trick 🤞

**Comment by Illia65** [comment]
Please respond to this comment

**Reply by AidenJamesYt** [reply]
?

**Comment by Illia65** [comment]
Response#1
